Resources/Prototypes/Entities/Objects/Specific/chemistry.yml

```
@@ -272,6 +272,7 @@
   - type: Sprite
     sprite: Objects/Specific/Chemistry/pills.rsi
     state: pill
+    scale: 0.5, 0.5
```

```suggestion
    scale: 0.35, 0.35
```
? Looked a little large in the preview.

---

I don't think the preview is scaled.